package.json

```diff
@@ -37,6 +37,7 @@
     "@babel/traverse": "^7.14.0",
     "@babel/types": "^7.0.0",
     "@jest/create-cache-key-function": "^29.7.0",
+    "@rushstack/node-core-library": "^5.7.0",
```

Comment:
I'm hesitant to bring in this dependency for a set of things that look like pretty thin wrappers around node apis. Do you think it could be avoided? What's the cost of not including it?

Reply:
using it for validating the loaded config against the json schema so we would otherwise need some dependency for doing that (e.g. `ajv` which this library uses under the hood).  also this dep should only be used in dev tools not anything at runtime